dm/syncer/data_validator.go

```diff
@@ -387,7 +391,12 @@ func (v *DataValidator) printStatusRoutine() {
 			speed := float64((currProcessedBinlogSize-prevProcessedBinlogSize)>>20) / interval.Seconds()
 			prevProcessedBinlogSize = currProcessedBinlogSize
 			prevTime = currTime
-
+			counts, err := v.getAllErrorCount(validatorStatusInterval)
```

Comment:
prefer use `validatorDmctlOpTimeout`, 1m is too long

Reply:
Will change it to 5s